Docs clarifications (#17)
raspy135/Qun-mk2: README.md

# Contact

- Email: info@nunomo.net
- Github: Open an issue if you have a GitHub account. It's the best place to submit bugs.

> - [Discord](https://discord.gg/kcvVTPkZVQ): Visit Nunomo's Discord server chat about Qun, read announcements, and talk about events.

Reviewer: ```suggestion
- [Discord](https://discord.gg/kcvVTPkZVQ): Visit Nunomo's Discord server to chat about Qun, read announcements, and talk about events.
```

- Social media links:
  - Instagram: [@nunomo1](https://www.instagram.com/nunomo1)
  - Twitter (Japanese): [@nunomo1](https://twitter.com/nunomo1)
  - YouTube: [nunomollc](https://youtube.com/@nunomollc)

# QUN mk2

*Qun mk2* is a small synthesizer with a powerful sound engine, sequencer, and looper.

![qun_mk2_cgi](./manual_images/qun_mk2_cgi.png)

## Links and Information

* [Online Shop](https://shop.nunomo.net/products/): Purchase your own from our website.

> * [Presets](https://github.com/raspy135/Qun-mk2/blob/main/sd_template.zip): Download tone presets and install (see under Load Preset). We constantly update the tone presets.

Reviewer: ```suggestion
* [Presets](https://github.com/raspy135/Qun-mk2/blob/main/sd_template.zip): Download and install tone presets  (see [Load Preset](#load-preset)). We constantly update the tone presets.
```

* [Issues](https://github.com/raspy135/Qun-synthesizer/issues): Report issues when you have problems or questions.
* [Firmware](./firmware): Firmware updates and instructions (see README.md).
* [CheatSheet](./cheatsheet.pdf): A printable cheat sheet listing buttons, device modes and sub-modes, shortcuts, and synth instrument parameters.

**We are looking for your feedback! For general impressions and feedback, please email them to info@nunomo.net or join Nunomo's Discord server!**

## Overview of the QUN mk2

Highlights:

Qun mk2 has a 2 oscillator virtual analog, FM, granular sound engine. The 2 oscillators can be used separately.

The sequencer is an 8-step based sequencer, and supports up to 32/64 steps per pattern. You can store 8 patterns for each preset bucket. And it has 8 buckets of tone preset + sequencer patterns. So you can quickly switch between 8 presets and 64 (8 x 8 pattern) patterns.

> It has a 3 track looper with 5 scenes, so you can swap 3 track x 5 scenes while playing. 

Reviewer: ```suggestion
It has a 3 track looper with 5 scenes, so you can swap 3 tracks x 5 scenes while playing. 
```

* Double click the button to enter relative mode. The value adjustment will be relative.
* For initial training, referring to the [Qun cheatsheet](./cheatsheet.pdf) can be useful. Also seeing the parameter name when changing its value is useful for training: turn on Parameter Caption mode in System2 menu, button 3.
* If you want to do fine value adjustment, you can use NO/OK buttons to change the value by 1. To do this, keep pressing one of 8 buttons and press NO or OK button.

Button | Primary function
------------ | -------------
SHIFT | Shift + [1-8] button or NO/OK button to select sub-mode 
[1-8] button | Select a parameter in the current sub-mode. Also show information with LEDs (see under LED Indicators).
NO | Select previous sequencer pattern
OK | Select next sequencer pattern
Dial | change parameters
SEQ PLAY | Start / Stop sequencer
REC + SEQ PLAY | Start sequencer live recording
MODE PLAY | Select Play mode
SYSTEM | Select System mode
PARAM | Select Parameter mode
REC | Looper status page
REC + LOOPER PLAY | Start looper recording
LOOPER PLAY | Start looper playing
LOOPER STOP | Stop looper playing
RST button on the base board | Reset the device

> 4 touch buttons at the bottom board | This can be used as a touch slider. See Touch Slider section for detail. 

Reviewer: ```suggestion
4 touch buttons at the bottom board | This can be used as a touch slider. See [Touch Slider](#touch-slider) section for detail. 
```

SHIFT + NO + OK | Sleep (Shutdown) the device. It will turn off most of the power consuming components like the main CPU and the screen. Press the RST button to restart the device.

## BASIC SHORTCUTS

Button | Function
------------ | -------------
SHIFT + MODE PLAY | Dump current preset to MIDI OUT. This is useful to transfer the current preset to another Qun / Qun mk2 synthesizer.
SHIFT + SYSTEM | All notes off

> SHIFT + PARAM | Initialize the preset. You can choose some templates: Blank, Sampler, EvenSlice, Scratch. Please see the "Initialize preset" section for detail. 

Reviewer: ```suggestion
SHIFT + PARAM | Initialize the preset. You can choose some templates: Blank, Sampler, EvenSlice, Scratch. Please see the [Initialize preset](#initialize-preset) section for detail. 
```

LOOPER PLAY + [1-5] + Dial | Copy Scene
REC + [1-3, 7,8] | Select recording track. A,B,C are mono tracks. Press 7, 8 for stereo recording. 7=A(Left)+B(Right), 8 = B+C. When you do stereo recording, you may want to set pan to left and right for the selected stereo tracks. 
PARAM + [1-3] | Mute track

> MODE PLAY + [1-8] | Recall preset bucket (see under Preset Bucket)

Reviewer: ```suggestion
MODE PLAY + [1-8] | Recall preset bucket (see under [Preset Bucket](#preset-bucket))
```

## Session

Qun automatically collects the current looper recordings, instrument and sequencer presets into a Session. Sessions can be saved and loaded. They are automatically numbered and can be manually named.

![session_diagram](./manual_images/session_diagram.png)

Session is the biggest data unit of the synth.
Session includes 8 preset buckets. Each bucket contains one preset with 8 pattern sequencer data.

There is no "new session" function. The synth always create a new session when booting. Reboot the device to create a new sesion.

Session button operations work in Parameter mode or Play mode

Button | Function
------------ | -------------
SHIFT + LOOPER PLAY | Show current session number
SHIFT + LOOPER PLAY + Turn Dial | Load session
NO or OK button while selecting a session | Scroll session page when you have more than 32 sessions 
SHIFT + REC | Save current session (Looper stop will perform session save when autosave is on)
SHIFT + REC (Long press) | Name current session

> SHIFT + REC + B[1-3] | Load WAV file to looper track A, B or C (See looper section for detail)

Reviewer: ```suggestion
SHIFT + REC + B[1-3] | Load WAV file to looper track A, B or C (See [Looper Overview](#looper-overview) section for details)
```

![‎qun2_diagram](./manual_images/‎qun2_diagram.png)

## Parameter Mode (Param)

To operate, press B[1-8] to choose a parameter and rotate dial to change its value. The parameter display area will temporarily invert. A parameter's position in the display corresponds to the visual button assignment:

```
 1  3  5  7
 2  4  6  8
```

E.g. button 1 chooses the wave SHAPE parameter when in the OSCILLATOR1,2 sub-mode.

### PRM:OSCILLATOR1,2

![osc1](./manual_images/osc1.png)

The oscillator is a hybrid of classic analog synth, FM, and granular.

Buttons | Description
-------|-------

> 1 | SHAPE. `Saw, Sine, S&H, Square, Triangle, W Noise, P Noise, FM, AUX`.<br> * AUX means audio input from LINE or MIC. <br> * Granular synth engine is connected to the AUX L channel when it's active. See Granular section for detail. 

Reviewer: ```suggestion
1 | SHAPE. `Saw, Sine, S&H, Square, Triangle, W Noise, P Noise, FM, AUX`.<br> * AUX means audio input from LINE or MIC. <br> * Granular synth engine is connected to the AUX L channel when it's active. See [Granular](#granular) section for detail. 
```

Button | Description
------ | ------
1 | VCF 4/2 POLE / NoLinear. Select VCF number of poles and linear or no linear. 2P/4P/2PNL/4PNL. Ladder filter algorithm will be used only when 4NL with Lowpass or Highpass filter. 
2 | VCF ENV Src. Applying Envelopes to the signal after VCF. You might want to set OSC1/2's Env sel to "ON" to utilize this parameter. This is useful only with Mono mode. 
3 | GLIDE. Smooth transition between notes. Works with Mono.
4 | BEND RANGE. Pitch bend range.
5 | OSC1/2 Keysync. This is a switch to connect MIDI input notes and oscillators. Y = Follow MIDI note pitch. N = Ignore MIDI note. You can still use FREQ1 and FREQ2 for modulation to use MIDI note pitch.
6 | VELOCITY SW. Envelope generators become velocity sensitive when it’s ON.

> 7 | MONO/QUAD/POLY (Voice mode). Long press to sync Oscillator parameters. See the Voice Mode section for details. 

Reviewer: ```suggestion
7 | MONO/QUAD/POLY (Voice mode). Long press to sync Oscillator parameters. See the [Voice Mode](#voice-mode) section for details. 
```

This means A modulates B, the result modulates C, the result modulates D.

Oscillator (ABC)’s frequency (multiple of frequency of D).
	When FM FREQ SNAP (In ENV3/4 Other sub-menu) is off, it can be adjusted to the exact frequency of harmonics (2,3,4,5..). If you want clean FM sound, turn FM FREQ SNAP on.

### PRM:ENV3/4 / Other

![fm2](./manual_images/fm2.png)

Button | Description
--------|--------

> 1 | 2ND FILTER. See the 2nd Filter section for details.

Reviewer: ```suggestion
1 | 2ND FILTER. See the [2nd Filter](#2nd-filter) section for details.
```

Button | Function
------------ | -------------
1 | Process samples.  Analyze will give better result with Time stretch. See below list for processing  detail 

> 2 | Slice Spread, half-steps. Please see below section for detail. 

Reviewer: ```suggestion
2 | Slice Spread, half-steps. Please see [Having Multiple Slices (Slice Spread)](#having-multiple-slices-slice-spread) section for detail. 
```
